### ISSUE TITLE
Compatibility Issues with Homebridge

### ISSUE THREAD

**Issue by jsiegenthaler** [ztalbot2000/homebridge-cmd4#56]
Homebridge reports:

[6/5/2020, 15:53:52] The plugin "homebridge-cmd4" defines 'homebridge' and/or 'hap-nodejs' in their 'dependencies' section, meaning they carry an additional copy of homebridge and hap-nodejs. This not only wastes disk space, but also can cause major incompatibility issues and thus is considered bad practice. Please inform the developer to update their plugin!

Can you fix this?

**Comment by ztalbot2000** [comment]
duplicate of #51

**Comment by ztalbot2000** [comment]
Hi,

   A long time ago, I removed hap-nodejs as a dependancy. As well as homebridge. I'd really like to put hap-nodejs back so that at run-time when I include homebridge/plugin-ui-utils I can check things like format types and ranges, without duplicating this in my current code.  Unfortunately including homebridge/plugin-ui-utils they have no way of accessing _api in server.js to get a run-time way of accessing formats/units ...

 I don't see this as a problem, but I do remember you wanting me to remove this dependancy, which I did at the time.  I was wondering if you had any thoughts on the matter.

Thanks,
John Talbot
ztalbot2000@gmail.com
